components/admin_console/team_channel_settings/channel/list/channel_list.tsx

```diff
@@ -237,6 +238,7 @@ export default class ChannelList extends React.PureComponent<ChannelListProps, C
                 filters.exclude_group_constrained = excludeGroupConstrained.value as boolean;
             }
             filters.team_ids = teamIds.value as string[];
+            trackEvent('admin_channels_page', 'filters_applied_to_channel_list');
```

Comment:
@thefactremains similar question here - do we want different events for each filter applied? Or is a generic `filters_applied_to_channel_list` fine?

Reply:
Discussed on mattermost decided to go with a different event for each filter for this and the other one